ts/src/kucoin.ts

```diff
@@ -3965,40 +3998,27 @@ export default class kucoin extends Exchange {
             'currency': currency['id'],
             'size': this.currencyToPrecision (code, amount),
         };
-        let method = undefined;
         const timeInForce = this.safeStringN (params, [ 'timeInForce', 'type', 'borrowStrategy' ], 'IOC');
-        let timeInForceRequest = undefined;
-        if (symbol === undefined) {
-            method = 'privatePostMarginBorrow';
-            timeInForceRequest = 'type';
-        } else {
+        if (isIsolated) {
+            if (symbol === undefined) {
+                throw new Error ('symbol is required when isolated margin');
```

Comment:
@sc0Vu we should not throw an error that generic, why not `BadRequest` for instance?

Reply:
@carlosmiei how about ArgumentsRequired error?

Reply:
@sc0Vu yeah it can be that as well